cgrok/pcreposix.c

```diff
@@ -57,6 +57,7 @@ previously been set. */
 #  define PCREPOSIX_EXP_DEFN __declspec(dllexport)
 #endif
 
+#define PCRE_STATIC
```

Comment:
Necessary on Windows to avoid the exported names getting mangled for linking against a DLL.